pennies/__init__.py

```diff
@@ -0,0 +1,62 @@
+import check50
+import check50.c
+
+@check50.check()
+def exists():
+    """pennies.c exists"""
+    check50.exists("pennies.c")
+
+@check50.check(exists)
+def compiles():
+    """pennies.c compiles"""
+    check50.c.compile("pennies.c", lcs50=True)
+
+@check50.check(compiles)
+def test28days1penny():
+    """28 days, 1 penny on day one yields $2684354.55"""
+    check50.run("./pennies").stdin("28").stdin("1").stdout("\$2684354.55\n", "$2684354.55\n").exit(0)
```

Comment:
If you want a string to match exactly (i.e. you don't want it to be interpreted as a regex), you can pass `regex=False` to `stdout`. This also helps prevent subtle bugs since this check would currently accept "$2684354X55" where "X" is any character (`.` is a special symbol in regexes that matches any character). With this change, the line becomes:

```python
check50.run("./pennies").stdin("28").stdin("1").stdout("$2684354.55\n", regex=False).exit(0)
```

Ditto for the rest of these.